.gitignore

```
@@ -1,2 +1,3 @@
 .idea/*
 .python-version
+rabbitmq/.DS_Store
```

```suggestion
.DS_Store
```
We want to ignore `.DS_Store` anywhere it pops up.